tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -7,12 +7,13 @@ class BugzillaLite(Base):
     _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
     _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
     _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, "a[href='/profile/']")
+    _button_logout = (By.CSS_SELECTOR, "a[href='/logout/']")
     _dashboard_navigator_locator = (By.ID, 'dashboardNav')
     _dashboard_login_locator = (By.ID, 'login')
```

Comment:
Nit: After taking a closer look to the DOM. I think we can also change the name of this locator. How about `_login_form_locator` ?